src/Transports.Subscriptions.WebSockets/WebSocketConnectionFactory.cs

```diff
@@ -14,26 +16,45 @@ public class WebSocketConnectionFactory<TSchema> : IWebSocketConnectionFactory<T
         private readonly IGraphQLExecuter<TSchema> _executer;
         private readonly IEnumerable<IOperationMessageListener> _messageListeners;
         private readonly IDocumentWriter _documentWriter;
+        private readonly IServiceScopeFactory _serviceScopeFactory;
 
-        public WebSocketConnectionFactory(ILogger<WebSocketConnectionFactory<TSchema>> logger,
+        [Obsolete]
+        public WebSocketConnectionFactory(
+            ILogger<WebSocketConnectionFactory<TSchema>> logger,
             ILoggerFactory loggerFactory,
             IGraphQLExecuter<TSchema> executer,
             IEnumerable<IOperationMessageListener> messageListeners,
             IDocumentWriter documentWriter)
+            : this(logger, loggerFactory, executer, messageListeners, documentWriter, null)
```

Comment:
Why not `NoopServiceScopeFactory.Instance` ?

Reply:
In any case, we delete it in the next major version.

Reply:
Because it’s private. Right.

Reply:
I could make it internal for that reason but it’s better scoped this way.

Reply:
Whatever.